scraper.rb

```diff
@@ -59,8 +59,14 @@ class MemberPage < Page
     url.to_s.split('/').last
   end
 
+  TITLE_RE = /^(Hon\.|Sen\.) /
+
   field :name do
-    box.xpath('.//th[text()="Name"]/following-sibling::td').text.tidy.sub('Hon. ', '')
+    name_with_title.gsub(TITLE_RE, '')
+  end
+
+  field :title do
+    $1 if name_with_title.match(TITLE_RE)
```

Comment:
Oh, "Avoid the use of Perl-style backrefs." - interesting. I've fixed with the house style in 0dacd1fd91a78126d